src/scrollable.directive.ts

```diff
@@ -99,7 +99,7 @@ export class NguiScrollableDirective {
     // detect the current environment
     let parentElStyle = window.getComputedStyle(parentEl);
     let scrollContainerEl = parentElStyle.overflow === 'auto' ?
-      parentEl : document.body;
+      parentEl : document.scrollingElement || document.documentElement;
```

Comment:
`document.documentElement` will not work on chrome.But `document.scrollingElement` works for both chrome & firefox.Anyway for safety will all types of browsers , `document.documentElement` is included.